content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,64 +1,61 @@
 ---
 reviewers:
 - sftim
-title: LimitRange
+- kbhawkey
+title: Limit Ranges
 content_template: templates/concept

```

Comment:
Hi, @nelvadas. I would rather not be listed as a reviewer, as I am fairly new to Limit Ranges. Is there a SIG group that should review the content?

Reply:
Sorry, updated the section accordingly. You are already doing an amazing review.